paparazzi-agent/src/main/java/app/cash/paparazzi/agent/AgentTestRule.kt

```diff
@@ -11,7 +10,6 @@ class AgentTestRule : TestRule {
     description: Description
   ) = object : Statement() {
     override fun evaluate() {
-      ByteBuddyAgent.install()
```

Comment:
This was moved into `InterceptorRegistrar` so it can be automatically called for both JUnit 4 & 5 tests.